src/qibo/gates/gates.py

```diff
@@ -1540,7 +1611,7 @@ def on_qubits(self, qubit_map):
         return gate
 
     def _dagger(self):
-        import numpy as np
+        import numpy as np  # pylint: disable=C0415
```

Comment:
Not very related to this PR, but this can also be a global import (to fix pylint instead of disabling).

Reply:
Done